srcpkgs/tbb/template

```diff
@@ -1,76 +1,52 @@
 # Template file for 'tbb'
 pkgname=tbb
-reverts="2021.5.0_1"
-version=2020.U3
-revision=2
-# version rewrite: 2017_U7 (upstream) => 2017.7 (xbps)
-_ver=${version/./_}
-build_style=gnu-makefile
-hostmakedepends="cmake"
+version=2021.11.0
+revision=1
+build_style=cmake
+configure_args="-DTBB_STRICT=OFF -DTBB_TEST=OFF"
+makedepends="libgomp-devel libhwloc-devel"
 short_desc="Intel Threading Building Blocks"
 maintainer="Andrea Brancaleoni <abc@pompel.me>"
 license="Apache-2.0"
-homepage="https://www.threadingbuildingblocks.org"
-distfiles="https://github.com/oneapi-src/oneTBB/archive/${_ver}.tar.gz"
-checksum=2103cc6238c935664f87680618f6684d57501d4a2fa8ea8f6c97ad6ff7dc722a
+homepage="https://oneapi-src.github.io/oneTBB/"
+changelog="https://raw.githubusercontent.com/oneapi-src/oneTBB/v${version}/RELEASE_NOTES.md"
+distfiles="https://github.com/oneapi-src/oneTBB/archive/refs/tags/v${version}.tar.gz"
+checksum=782ce0cab62df9ea125cdea253a50534862b563f1d85d4cda7ad4e77550ac363
+make_check=no  # checks fail upstream on i686
 
 case "$XBPS_TARGET_MACHINE" in
-	x86_64*)
-		make_build_args="arch=intel64"
-		;;
-	i686*)
-		make_build_args="arch=ia32"
-		;;
-	armv7l*)
-		make_build_args="arch=armv7"
-		;;
-	aarch64*)
-		make_build_args="arch=arm64"
-		;;
-	ppc64*)
-		make_build_args="arch=ppc64"
-		;;
-	ppc*)
-		make_build_args="arch=ppc32"
-		;;
-	*)
-		# make sure to fall back instead of letting host uname control it
-		make_build_args="arch=generic"
-		;;
+	x86_64*|aarch64*)
+		build_options_default+=" -DTBB4PY_BUILD=ON"
```

Comment:
This is broken https://github.com/void-linux/void-packages/pull/48993

Reply:
yes this is a mistake